subprojects/language-native/src/main/java/org/gradle/language/nativeplatform/internal/toolchains/DefaultToolChainSelector.java

```diff
@@ -66,13 +71,15 @@ public DefaultToolChainSelector(ModelRegistry modelRegistry) {
             }
         }
 
-        // TODO - use a better name for the platforms, rather than "current"
+        // TODO - use a better name for the platforms, rather than "host"
```

Comment:
Perhaps this should just be `targetMachine.getName()` or some combination of `targetMachine` and `operatingSystemFamily`.

Reply:
I think this could go away with a move away from NativePlatform or when we support cross compilation (doing something like your suggestion).